README.md

```diff
@@ -31,7 +31,7 @@ $ composer require rubix/ml
 Read the latest documentation [here](https://docs.rubixml.com/en/latest/).
 
 ## Getting Started
-If you are new to machine learning, we recommend the [basic introduction](basic-introduction.md) to get started. From there, you can browse the [example projects](https://github.com/RubixML) and tutorials listed below which range from beginner to advanced skill level.
+If you are new to machine learning, we recommend the [basic introduction](docs/basic-introduction.md) to get started. From there, you can browse the [example projects](https://github.com/RubixML) and tutorials listed below which range from beginner to advanced skill level.
```

Comment:
Thank you for catching this @nterray and thank you for the PR

We actually intended to have this link point to https://docs.rubixml.com/en/latest/basic-introduction.html

Would you be able to make that change for us real quick?

Reply:
Sure :)